packages/drivers/local-driver/src/localDocumentService.ts

```diff
@@ -17,14 +17,14 @@ import { LocalDeltaStorageService, LocalDocumentDeltaConnection } from ".";
 export class LocalDocumentService implements api.IDocumentService {
     /**
      * @param localDeltaConnectionServer - delta connection server for ops
-     * @param tokenProvider - token provider with a single token
+     * @param tokenProvider - token provider
      * @param tenantId - ID of tenant
      * @param documentId - ID of document
      */
     constructor(
         public readonly resolvedUrl: api.IResolvedUrl,
         private readonly localDeltaConnectionServer: ILocalDeltaConnectionServer,
-        private readonly tokenProvider: socketStorage.TokenProvider,
+        private readonly tokenProvider: socketStorage.ITokenProvider,
```

Comment:
Note - in case of SPO, there are two providers - one for storage, one for web-socket.
I do not know why it's the case though

Reply:
I see, you are putting responsibility to fetch both tokens from same provider